ipu-plugin/pkg/ipuplugin/lifecycleservice.go

```diff
@@ -968,12 +986,40 @@ func skipIMCReboot() (bool, string) {
 
 }
 
+// wait for the expected number of ACC APFs to get initialized.
+// Per testing, after kernel boots, it takes around 40 secs for 48 APFs
+// to get initialized. This api waits for 1 sec per APF.
+func waitForAccApfsInit() error {
```

Comment:
Basically, we are adding 40 seconds to ipu-plugin init. Is there anyway we can optimize the APF creation?. Please check with Zhenning on this topic.
Lets create a separate JIRA to optimize the init flow.